src/main/java/tickbot/exception/CorruptedDataException.java

```diff
@@ -0,0 +1,12 @@
+package tickbot.exception;
+
+/**
+ * The exception to represent that the data storage is corrupted.
+ */
+public class CorruptedDataException extends Exception {
+    private static final long serialVersionUID = 1L;
```

Comment:
Should final variables be capitalized?

Reply:
No, because this is the explicitly-defined `serialVersionUID` required when implementing `java.io.Serializable`. See [](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/Serializable.html).